packages/core/src/effection.ts

```diff
@@ -6,7 +6,7 @@ const MAJOR_VERSION = version.split('.')[0];
 const GLOBAL_PROPERTY = `__effectionV${MAJOR_VERSION}`;
 
 function createRootTask() {
-  let task = createTask(undefined, { ignoreChildErrors: true });
+  let task = createTask(undefined, { ignoreChildErrors: true, labels: { name: 'root' } });
```

Comment:
Two hunches:

From a purely branding perspective, `effection` feels hotter

```suggestion
  let task = createTask(undefined, { ignoreChildErrors: true, labels: { name: 'effection' } });
```

Also, shouldn't we just omit the root task from the tree?

Reply:
> From a purely branding perspective, `effection` feels hotter

Given that the root task is accessed through `Effection.root`, I feel like `root` should be in there somewhere. Maybe something like `effectionRoot`, but that feels awkward to me.

> Also, shouldn't we just omit the root task from the visualization?

That is an option too, but it might be instructive to see it, I think?

Reply:
After discussion, we can leave as-is, in other word, we label it after its name in the API, and we add an option to the debugger to show the root task.